extensions/agent/client/src/dialogs/createJobDialog.ts

```diff
@@ -188,7 +188,7 @@ export class CreateJobDialog {
 			}).component();
 
 			this.newStepButton.onDidClick((e)=>{
-				let stepDialog =new CreateStepDialog(this.model.ownerUri, '', '', this.model);
+				let stepDialog = new CreateStepDialog(this.model.ownerUri, '', '', 1, this.model);
```

Comment:
I'm not sure how to pass a stepId in the job dialog, so I put a default value of 1 right now. The logic will change in the create job dialog side.